django/db/backends/oracle/base.py

```diff
@@ -383,8 +383,8 @@ class FormatStylePlaceholderCursor:
     """
     charset = 'utf-8'
 
-    def __init__(self, connection):
-        self.cursor = connection.cursor()
+    def __init__(self, connection, *args, **kwargs):
+        self.cursor = connection.cursor(*args, **kwargs)
```

Comment:
It turns out that you can not pass args/kwargs to the `cursor()` on Oracle (see [[1]](http://cx-oracle.readthedocs.io/en/latest/connection.html#Connection.cursor)). Sorry for misleading, please remove args/kwargs from the `FormatStylePlaceholderCursor.__init__` and add an comment in the `create_cursor` on Oracle.